.travis.yml

```diff
@@ -7,11 +7,14 @@ rvm:
 gemfile:
   - Gemfile.rails42
   - Gemfile.rails50
+  - Gemfile.rails51
 
 matrix:
   exclude:
     - gemfile: Gemfile.rails50
       env: DB=postgresql
+    - gemfile: Gemfile.rails51
+      env: DB=postgresql
```

Comment:
This has the same test failure as rails 5.0 for an unintended serialization format change